### PR TITLE
add standard deviation doc

### PR DESCRIPTION
**Pull request by Mimetis**
The following changes are proposed:

* add documentation about extended stats (standard deviation)

### CODE REVIEW

**Review by dupuyjs — APPROVED** [review]

File: docs/visualization/metrics.md [hunk 1]
```diff
@@ -79,6 +79,40 @@ A multi-value metrics aggregation that computes stats over numeric values extrac
...
+The Standard Deviation Bounds values are calculated internally by K2Bridge:
+
+```
+Lower Deviation (Population or Sampling) = Average Value - (Standard Deviation (Population or Sampling) * Sigma)
+Upper Deviation (Population or Sampling) = Average Value + (Standard Deviation (Population or Sampling) * Sigma)
+```
+
+The `Sigma` value controls controls how many standard deviations +/- from the mean should be displayed.
+
+> As reference, you can check the Elasticsearch implementation used to calcul bounds here : [getStdDeviationBound()](https://github.com/elastic/elasticsearch/blob/0699c9351f1439e246d408fd6538deafde4087b6/server/src/main/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStats.java#L187-L194)
```
Comment (eladiw) [inline docs/visualization/metrics.md:97]:
```suggestion
> As reference, you can check the Elasticsearch implementation used to calculate bounds here : [getStdDeviationBound()](https://github.com/elastic/elasticsearch/blob/0699c9351f1439e246d408fd6538deafde4087b6/server/src/main/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStats.java#L187-L194)
```

File: docs/visualization/metrics.md [hunk 2]
```diff
@@ -79,6 +79,40 @@ A multi-value metrics aggregation that computes stats over numeric values extrac
...
+| extend ['f4674a57-bb73-4916-8a3f-c0bb65ee4c0d'] = true;
+
+let _summarizablemetrics = _extdata
+| summarize ['1%extended_stats%2']=pack('count', count(),'min', min(['AvgTicketPrice']),'max', max(['AvgTicketPrice']),'avg', avg(['AvgTicketPrice']),'sum', sum(['AvgTicketPrice']),'sum_of_squares', sum(pow(['AvgTicketPrice'], 2)),'variance_population', variancep(['AvgTicketPrice']),'variance_sampling', variance(['AvgTicketPrice']),'std_deviation_population', stdevp(['AvgTicketPrice']),'std_deviation_sampling', stdev(['AvgTicketPrice'])),count() by ['f4674a57-bb73-4916-8a3f-c0bb65ee4c0d'];
+
+(_summarizablemetrics
+| project-away ['f4674a57-bb73-4916-8a3f-c0bb65ee4c0d']
+| as aggs);
+```
+
```
Comment (eladiw) [inline docs/visualization/metrics.md:115]: At first your describe the role of the 'sigma' value, but in the adx query you can't see where the  std deviation is tied to it. I would add a paragraph on the fact this value is part of the column name + that it is later calculated (after the adx query is executed)

**Review by eladiw — CHANGES_REQUESTED** [review]
see my comment regarding the sigma value

File: docs/visualization/metrics.md (hunk 2 above)
Comment (Mimetis) [reply docs/visualization/metrics.md:138]: Hey @eladiw 
I've made an update on the doc, and fix a small issue on double parsing !
Thank for the comment !

**Review by AsafMah — APPROVED** [review]

**Review by eladiw — APPROVED** [review]

**Pull request merged**